lib/generators/decidim/templates/initializer.rb

```diff
@@ -6,8 +6,8 @@
   config.mailer_sender = "change-me@domain.org"
   config.authorization_handlers = [ExampleAuthorizationHandler]
 
-  # Uncomment this lines to set your preferred locales
-  # config.available_locales = [:en, :ca, :es]
+  # Change this line to set your preferred locales
+  config.available_locales = [:en, :ca, :es]
```

Comment:
Yep, I'd change the values in the seeds until we start working on not making all the languages compulsory